projects/app-lob/src/app/grid-finjs/main.component.ts

```diff
@@ -20,7 +20,7 @@ export class FinJSDemoComponent implements OnDestroy {
     public darkTheme = false;
 
     public properties = ['price', 'country'];
-    public chartData: Stock[] = [];
+    public chartData: any[] = [];

```

Comment:
Good practice is to always type, and type to `any` only if really really needed. Not sue if this is the case.